### DISCUSSION TITLE
Turnkey solution for keygen.sh-integrated web shop?

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#27] [Q&A] [answered]
I'm looking for an easy-to-set-up way for selling my keygen.sh-licensed software, e.g., a Wordpress-like webshop or so. What's everyone using nowadays?

**Comment by ezekg** [comment, marked as answer]
No turn-key solution right now. But [our Zapier integration](https://zapier.com/apps/keygen/integrations) can get you most of the way there using something like Stripe Checkout.

**Reply by nschloe** [reply]
Yup, that's what I'm looking at right now. Thanks!
